### PR TITLE
fixed some problems with tasks and improved some loggings

### PR DESCRIPTION
**Pull request by sveneberth**
Fixed db.Key bug in l.228

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -178,7 +179,7 @@ def deferred(self, *args, **kwargs):
 				raise errors.RequestTimeout()  # Task-API should retry
 		elif cmd == "unb":
 			if not funcPath in _deferedTasks:
-				logging.error("Ive missed a defered task! %s(%s,%s)" % (funcPath, str(args), str(kwargs)))
+				logging.error("Ive missed a deferred task! %s(%s,%s)", funcPath, args, kwargs)
```
Comment (phorward) [inline tasks.py:182]: Who is Ive? ;-)

File: tasks.py [hunk 2]
```diff
@@ -331,7 +331,7 @@ def mkDefered(func, self=__undefinedFlag_, *args, **kwargs):
 			if req:
 				req.pendingTasks.append(task)  # < This property will be only exist on development server!
 			else:
-				# Warmup request or something - we have to call it now as we can't deferr it :/
+				# Warmup request or something - we have to call it now as we can't deffer it :/
```
Comment (phorward) [inline tasks.py:334]: defer => https://dict.leo.org/german-english/defer

File: tasks.py (hunk 1 above)
Comment (sveneberth) [reply tasks.py:182]: ?

Comment (sveneberth) [reply tasks.py]: Idk. Maybe a friend of @tsteinruecken :)

**Pull request merged**